src/display/api.js

```diff
@@ -1192,9 +1196,15 @@ var PDFWorker = (function PDFWorkerClosure() {
 //    PDFJS.fakeWorkerFilesLoadedCapability.resolve();
 //#endif
 //#if PRODUCTION && !SINGLE_FILE
-//    Util.loadScript(PDFJS.workerSrc, function() {
-//      PDFJS.fakeWorkerFilesLoadedCapability.resolve();
+//  if (fakeWorkerLoader) {
```

Comment:
Since `fakeWorkerLoader` is only included (through `../src/frameworks.js`) in `GENERIC` builds, won't this throw an error in other build types?

(Also, given that the preprocessor should now support chained `#if`s, it might be possible to simplify these preprocessor directives a bit.)